Examples/Sources/HoverExample/HoverApp.swift

```diff
@@ -0,0 +1,54 @@
+import DefaultBackend
+import Foundation
+import SwiftCrossUI
+
+#if canImport(SwiftBundlerRuntime)
+    import SwiftBundlerRuntime
+#endif
+
+@main
```

Comment:
Add `@HotReloadable` to the app, and `#hotReloadable { ... }` around the root `VStack` to support Swift Bundler hot reloading.